tst/EnergyPlus/unit/PollutionModule.unit.cc

```diff
@@ -468,63 +468,63 @@ TEST_F(EnergyPlusFixture, PollutionModule_TestOutputVariables)
-        EXPECT_EQ(format("Site:Environmental Impact {} SO2 Emissions Mass", fuelTypeNames[i]),
+        EXPECT_EQ(format("Site:Environmental Impact {} SO2 Emissions Mass", fuelName),
                   state->dataOutputProcessor->RVariableTypes(i * 17 + 7).VarName);
-        EXPECT_EQ(format("Site:Environmental Impact {} PM Emissions Mass", fuelTypeNames[i]),
+        EXPECT_EQ(format("Site:Environmental Impact {} PM Emissions Mass", fuelName),
                   state->dataOutputProcessor->RVariableTypes(i * 17 + 8).VarName);
-        EXPECT_EQ(format("Site:Environmental Impact {} PM10 Emissions Mass", fuelTypeNames[i]),
+        EXPECT_EQ(format("Site:Environmental Impact {} PM10 Emissions Mass", fuelName),
                   state->dataOutputProcessor->RVariableTypes(i * 17 + 9).VarName);
-        EXPECT_EQ(format("Site:Environmental Impact {} PM2.5 Emissions Mass", fuelTypeNames[i]),
+        EXPECT_EQ(format("Site:Environmental Impact {} PM2.5 Emissions Mass", fuelName),
                   state->dataOutputProcessor->RVariableTypes(i * 17 + 10).VarName);
-        EXPECT_EQ(format("Site:Environmental Impact {} NH3 Emissions Mass", fuelTypeNames[i]),
+        EXPECT_EQ(format("Site:Environmental Impact {} NH3 Emissions Mass", fuelName),
                   state->dataOutputProcessor->RVariableTypes(i * 17 + 11).VarName);
-        EXPECT_EQ(format("Site:Environmental Impact {} NMVOC Emissions Mass", fuelTypeNames[i]),
+        EXPECT_EQ(format("Site:Environmental Impact {} NMVOC Emissions Mass", fuelName),
                   state->dataOutputProcessor->RVariableTypes(i * 17 + 12).VarName);
-        EXPECT_EQ(format("Site:Environmental Impact {} Hg Emissions Mass", fuelTypeNames[i]),
+        EXPECT_EQ(format("Site:Environmental Impact {} Hg Emissions Mass", fuelName),
                   state->dataOutputProcessor->RVariableTypes(i * 17 + 13).VarName);
-        EXPECT_EQ(format("Site:Environmental Impact {} Pb Emissions Mass", fuelTypeNames[i]),
+        EXPECT_EQ(format("Site:Environmental Impact {} Pb Emissions Mass", fuelName),
                   state->dataOutputProcessor->RVariableTypes(i * 17 + 14).VarName);
-        EXPECT_EQ(format("Site:Environmental Impact {} Water Consumption Volume", fuelTypeNames[i]),
+        EXPECT_EQ(format("Site:Environmental Impact {} Water Consumption Volume", fuelName),
                   state->dataOutputProcessor->RVariableTypes(i * 17 + 15).VarName);
-        EXPECT_EQ(format("Site:Environmental Impact {} Nuclear High Level Waste Mass", fuelTypeNames[i]),
+        EXPECT_EQ(format("Site:Environmental Impact {} Nuclear High Level Waste Mass", fuelName),
                   state->dataOutputProcessor->RVariableTypes(i * 17 + 16).VarName);
-        EXPECT_EQ(format("Site:Environmental Impact {} Nuclear Low Level Waste Volume", fuelTypeNames[i]),
+        EXPECT_EQ(format("Site:Environmental Impact {} Nuclear Low Level Waste Volume", fuelName),
```

Comment:
I could imagine simplifying this slightly to an array of strings and then just looping over it.  It wouldn't really clean it up too much, so no need here.